### ISSUE TITLE
Docs: [Scaling with Redis Cluster] `cluster-slave-validity-factor` does not appear in the latest redis configuration

### ISSUE THREAD

**Issue by Dup4** [redis/redis-doc#2353]
- [Scaling with Redis Cluster](https://redis.io/docs/management/scaling/#redis-cluster-configuration-parameters:~:text=cluster%2Dslave%2Dvalidity%2Dfactor%20%3Cfactor%3E)

https://github.com/redis/redis/blob/7997874f4d2be9da1e26c77804569b0057c1e0a2/redis.conf#L1647

`cluster-slave-validity-factor` appears in this document, but in the latest `redis.conf` we don't see this parameter, only a more similar parameter name (`cluster-replica-validity-factor`), so doesn't our documentation need to be updated?

**Comment by itamarhaber** [comment]
Yep - thanks for catching that, would you like to submit a merge request?

**Comment by Dup4** [comment]
> Yep - thanks for catching that, would you like to submit a merge request?


OK, I can try to provide a PR.

**Comment by enjoy-binbin** [comment]
@Dup4 
thanks for catching that. can you also try to scan the doc and finding other s-word conf?
the doc may be outdated in many places
